### ISSUE TITLE
RoBERTaMultiChoice does not work with `roberta-large`

### ISSUE THREAD

**Issue by yuchenlin** [huggingface/transformers#2782]
# 🐛 Bug

## Information

Model I am using (Bert, XLNet ...):
**roberta-large**
Language I am using the model on (English, Chinese ...):
English 
The problem arises when using:
* [* ] the official example scripts: 
https://github.com/huggingface/transformers/tree/6c1b23554f8bb5b5e1f6c80969acab764c755678/examples#multiple-choice
* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [ *] an official GLUE/SQUaD task: **SWAG**
* [ ] my own task or dataset: (give details below)

## To reproduce

just follow the example code of running swag dataset but with `robert-large` instead of `roberta-base` (which works well)

```
export SWAG_DIR=~/swagaf-master/data/
python ./examples/run_multiple_choice.py \
--model_type roberta \
--task_name swag \
--model_name_or_path roberta-large \
--do_train \
--do_eval \
--do_lower_case \
--data_dir $SWAG_DIR \
--learning_rate 5e-5 \
--num_train_epochs 3 \
--max_seq_length 80 \
--output_dir tmp/swag_base \
--per_gpu_eval_batch_size=16 \
--per_gpu_train_batch_size=16 \
--gradient_accumulation_steps 2 \
--overwrite_output
```

And it will say:

02/08/2020 00:46:23 - INFO - transformers.modeling_utils -   **Weights of RobertaForMultipleChoice not initialized from pretrained model:** ['classifier.weight', 'classifier.bias']

02/08/2020 00:46:23 - INFO - transformers.modeling_utils -   **Weights from pretrained model not used in RobertaForMultipleChoice:** ['lm_head.bias', 'lm_head.dense.weight', 'lm_head.dense.bias', 'lm_head.layer_norm.weight', 'lm_head.layer_norm.bias', 'lm_head.decoder.weight']

Consequently, the script is to learn a model from the scratch instead of fine-tuning pre-trained roberta-large.

## Expected behavior

It should load the pre-trained weights of roberta-large model.

## Environment info

     
- `transformers` version: 2.4.1
- Platform: linux
- Python version: 3.7
- PyTorch version (GPU?): GPU
- Tensorflow version (GPU?): n/a
- Using GPU in script?: no
- Using distributed or parallel set-up in script?: no

**Comment by yuchenlin** [comment]
hi @aaugustin , would you please help take a look at this? Thank you so much!

**Comment by yuchenlin** [comment]
I also found a similar problem with `bert-large-uncase-masked-whole-word`

**Comment by vincentlux** [comment]
@yuchenlin May I ask if you figured out the bug?
